src/components/RelocationBox/RelocationBox.jsx

```diff
@@ -0,0 +1,393 @@
+  // This method is called when the save button is clicked under pending changes
+  const updateBoxInfo = async stat => {
+    await FYABackend.put('/boxHistory/update', {
+      boxID,
+      status: stat,
+      boxHolderName: boxHolderNameState,
+      boxHolderEmail: boxHolderEmailState,
+      zipCode: zipCodeState,
+      generalLocation: generalLocationState,
+      message: messageState,
+      messageStatus: messageStatusState,
+      launchedOrganically: launchedOrganicallyState,
+    });
+
+    const requests = [fetchBoxes('under review', false), fetchBoxes('pending changes', false)];
+    await Promise.all(requests);
+  };
+
+  // A function that approves a relocation box submission and updates the backend state accordingly and then refetches all boxes (boxes can be approved from any tab)
+  const approveRelocationBoxFromUR = async id => {
+    await FYABackend.put('/boxHistory/update', {
+      boxID,
+      status,
+      boxHolderName: boxHolderNameState,
+      boxHolderEmail: boxHolderEmailState,
+      zipCode: zipCodeState,
+      generalLocation: generalLocationState,
+      message: messageState,
+      messageStatus: messageStatusState,
+      launchedOrganically: launchedOrganicallyState,
+    });
+    await FYABackend.put('/boxHistory/approveBox', {
+      boxID: id,
+    });
+    const requests = [
+      fetchBoxes('under review', false),
+      fetchBoxes('pending changes', false),
+      fetchBoxes('evaluated', false),
+    ];
+    await Promise.all(requests);
+  };
+
+  return (
+    <ChakraProvider>
+      <div
+        // Conditional classes for approved/pending changes/rejected boxes to determine background coloring
+        className={`${styles.box}
+        ${status === 'evaluated' && approved ? styles['box-approved'] : ''}
+        ${status === 'evaluated' && approved === false ? styles['box-rejected'] : ''}
+        ${status === 'pending changes' ? styles['box-pending'] : ''}`}
+      >
+        <Accordion allowToggle>
+          <AccordionItem>
+            <h3>
+              <AccordionButton className={styles['accordian-button']}>
+                {/* Relocation box icon */}
+                <div className={styles['picture-div']}>
+                  <img src={RelocateBoxIcon} alt=" " />
+                </div>
+                {/* Box Number & date */}
+                <div className={styles['title-div']}>
+                  <p className={styles.title}>
+                    <p className={styles['box-number']}>Box #{boxID}</p>
+                    {date}
+                  </p>
+                </div>
+                <div className={styles['arrow-button']}>
+                  <AccordionIcon />
+                </div>
+              </AccordionButton>
+            </h3>
+            {/* Box picture */}
+            <AccordionPanel pb={4} className={styles['accordion-panel']}>
```

Comment:
I know this may sound scary (and we can do this in another PR too), but any way we can change this to use react hook forms? I noticed that we don't have any error checking for these fields (especially zip code) and we don't check if the required fields are missing.

Reply:
On second thought, RHF may be overkill for this. For now, let's make zip code a required field and use isValidZip() from the utils file to check that the zipcode is valid.

Reply:
Hm yeah that's fine, we can convert to RHF and add error checking in a separate PR.